lib/NativeCall.pm6

```diff
@@ -403,7 +390,7 @@ sub cglobal($libname, $symbol, $target-type) is export is rw {
     Proxy.new(
         FETCH => -> $ {
             nqp::nativecallglobal(
-                nqp::unbox_s(guess_library_name($libname)),
+                nqp::unbox_s(guess_library_name($libname).IO.absolute),
```

Comment:
If that is true I'd say there are bigger problems, like a method named "guess_library_name" trying to do magic, or indexing %lib on such return values